From 843f7db388e3cbb39f0d479845cf7b8c204a49ff Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Mon, 13 May 2013 18:40:54 +0900
Subject: [PATCH 5/9] ext/dfbvideosink: fix the incorrect conversion from pixel
 to byte in NV12

pixel2byte() is used to figure out a stride of image data.
When the color format is NV12, the stride in pixel is the same as in byte.
No need to convert the value.
---
 ext/directfb/dfbvideosink.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ext/directfb/dfbvideosink.c b/ext/directfb/dfbvideosink.c
index a99f9b3..046183a 100644
--- a/ext/directfb/dfbvideosink.c
+++ b/ext/directfb/dfbvideosink.c
@@ -611,7 +611,6 @@ pixel2byte (gint pixels, DFBSurfacePixelFormat dfbfmt)
 {
   switch (dfbfmt) {
     case DSPF_NV12:
-      return pixels * 3 / 2;
     case DSPF_NV16:
     case DSPF_YV12:
       return pixels;
-- 
1.8.1.2

